src/scene/graph-node.js

```diff
@@ -254,7 +254,7 @@ class GraphNode extends EventHandler {
      */
     constructor(name = 'Untitled', app) {
         super();
-        this._app = app;
+        this._appRef = app;
```

Comment:
Do we need to assign _appRef to the cloned copy when cloning in `cloneInternal`?

Reply:
Sounds like it should yes